docs/jwst/pipeline/calwebb_coron3.rst

```diff
@@ -121,11 +123,17 @@ ASN file, e.g. "jw86073-a3001_t001_nircam_f140m-maskbar_psfstack.fits."
 :Data model: `~jwst.datamodels.QuadModel`
 :File suffix: _psfalign
 
-For each science target exposure, all of the reference PSF images in the
-"_psfstack" product are aligned to each science target integration and saved to
-a 4D "_psfalign" product by the :ref:`align_refs <align_refs_step>` step. The output file
-name is exposure-based, with the addition of the associated candidate ID, e.g.
-"jw8607342001_02102_00001_nrcb3_a3001_psfalign.fits."
+For each science target exposure, all of the reference PSF images in
```

Comment:
Oh wait - This info should go into the description section of the `align_refs` step doc, rather than here in the `calwebb_coron3` pipeline doc. https://jwst-pipeline.readthedocs.io/en/latest/jwst/align_refs/description.html
That's why the descriptions here are so terse: there's more detailed info in the docs for each step.

Reply:
Maybe I'm just very confused. I thought that the html was generated from the rst files with sphinx.

Reply:
No, you're not confused, just my message was confusing. The rst file that needs updating is https://github.com/spacetelescope/jwst/blob/master/docs/jwst/align_refs/description.rst